### PR TITLE
Issue 3

### LINKED ISSUE #3: Support for Multiple response types

### PR DESCRIPTION
**Pull request by ashwini-desai**
Fixes #3

### CODE REVIEW

**Comment by niraj8** [comment]
@ashwini-desai Looks good👍

File: src/main/kotlin/apifi/codegen/Non200ResponseHandler.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package apifi.codegen
+
+import org.apache.http.HttpStatus
+
+object Non200ResponseHandler {
+
+    private val allExceptionDetailsHolder = listOf<ExceptionDetailsHolder>(
```
Comment (ashwini-desai) [inline src/main/kotlin/apifi/codegen/Non200ResponseHandler.kt]: I could not think of better approach, please suggest alternate for this class if any.

**Comment by ashwini-desai** [comment]
> @ashwini-desai Looks good👍

File: src/main/kotlin/apifi/codegen/Non200ResponseHandler.kt (hunk 1 above)
Comment (bhimsenp) [reply src/main/kotlin/apifi/codegen/Non200ResponseHandler.kt]: ErrorResponseHandler??

**Review by bhimsenp — APPROVED** [review]

**Pull request merged**